test/units/modules/network/fortios/test_fortios_cmdb_set.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2017 Fortinet, Inc.
+#
+# Author: Miguel Angel Munoz (magonzalez at fortinet.com) and
+#         Nicolas Thomas (nthomas at fortinet.com)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import sys
+from nose.plugins.skip import SkipTest
```

Comment:
We'll also need to switch to [pytest](https://docs.pytest.org/en/latest/) for iunit tests as documented [here](https://docs.ansible.com/ansible/devel/dev_guide/testing_units.html#structuring-unit-tests). The Ansible Testing Community Group is attempting to standardize tests moving forward. Thank you!